src/routes/api/sections.ts

```diff
@@ -1,8 +1,10 @@
 import { Router } from 'express';
 import querySectionRoute from './sections/querySection';
+import createSectionRoute from './sections/createSection';
 
 const router = Router();
 
-router.post('/:id', querySectionRoute);
+router.get('/', querySectionRoute);
```

Comment:
I changed this query api to get, but just noticed on the api doc that this is designed to be a post, I wonder what you guys think, which would be better

Reply:
GET is fine